test/createStore.spec.ts

```diff
@@ -505,6 +505,7 @@ describe('createStore', () => {
 
   it('throws if action type is missing', () => {
     const store = createStore(reducers.todos)
+    // @ts-ignore
     expect(() => store.dispatch({})).toThrow(
```

Comment:
TypeScript won't let you call `store.dispatch` with a missing action type, now that the `store` is typed properly; thus, we need to `// @ts-ignore` this line to allow the test to run w/o compilation errors.